operators/src/adapters/raster_subquery/raster_subquery_reprojection.rs

```diff
@@ -144,6 +116,98 @@ where
     }
 }
 
+fn projected_coordinate_grid_parallel(
+    pool: &ThreadPool,
+    tile_info: TileInformation,
+    out_srs: SpatialReference,
+    in_srs: SpatialReference,
+    valid_out_area: &SpatialPartition2D,
+) -> Result<Grid2D<Option<Coordinate2D>>> {
+    const OVERPROVISIONING: usize = 1; // split the work into multiple chunks per thread.
```

Comment:
Why would you want to do this?

Reply:
it is usefull when there is a large ddifference of work done in chunks. Currently we have as many chunks as we  have threads. So the runtime ultimately depends on the slowest task.

Reply:
i removed it